prod/westeurope/notifications/functions_pushnotif/function_app/terragrunt.hcl

```
@@ -85,6 +85,9 @@ inputs = {
 
     SLOT_TASK_HUBNAME = "ProductionTaskHub"
 
+    // activity default retry attempts
+    RETRY_ATTEMPT_NUMBER = 10
```

why different values?

---

I think staging should fail faster